src/chrome/content/rules/Jobvite.xml

```diff
@@ -1,8 +1,5 @@
 
 <!--
-Disabled by https-everywhere-checker because:
-Non-2xx HTTP code: http://web.jobvite.com/css/mktLPSupportCompat.css (200) => https://na-aba.marketo.com/css/mktLPSupportCompat.css (403)
-Non-2xx HTTP code: http://web.jobvite.com/rs/jobvite/images/jobvite-logo-gunmetal.png (200) => https://na-aba.marketo.com/rs/jobvite/images/jobvite-logo-gunmetal.png (403)
 
 	Nonfunctional subdomains:
```

Comment:
Could you update the comment for `web`?
Please also remove the `default_off` attribute to re-enable the ruleset.